<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>lecture 62 : finally clauses</title>
</head>
<body>
    finally clauses work with any cases : try or catch run or not. complete or not It will work

    <script>
        try {
            let num = 34;
            console.log(nu2m) // error accured
        }

        catch(err){
            console.log("there is some error") // this code run now
            console.log(erfr) // here some error , Next code not run
        }

        
        finally{
            console.log("working with any cases") //work in any cases
        }
        console.log("im not running") // bcz catch has some error
    </script>
</body>
</html>